README.md

```diff
@@ -487,7 +487,7 @@ interface Foo {
 }
 ```
 
-### Key Definition
+### Dynamic Property Keys
```

Comment:
The issue about "dynamic" property keys was #97 and it's not covered by this PR.